src/components/notebook-utils.js

```diff
@@ -68,9 +68,13 @@ export const analysisNameInput = ({ inputProps, ...props }) => h(ValidatedInput,
   }
 })
 
-export const tools = { rstudio: { label: 'RStudio', ext: 'Rmd' }, jupyter: { label: 'Jupyter', ext: 'ipynb' } }
+export const tools = {
+  RStudio: { label: 'RStudio', ext: 'Rmd', imageIds: ['terra-jupyter-bioconductor', 'RStudio', 'terra-jupyter-bioconductor_legacy'], defaultImageId: 'terra-jupyter-bioconductor' },
```

Comment:
This is an object holding constants that I've expanded here. Its proven very useful, used here https://github.com/DataBiosphere/terra-ui/pull/2544/files#diff-9d805f7072dde93f4df88c56002a65dc9e585b734281cd957426d0cbada83ff2R500 and here https://github.com/DataBiosphere/terra-ui/pull/2544/files#diff-9d805f7072dde93f4df88c56002a65dc9e585b734281cd957426d0cbada83ff2R481